aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/IntegrationTests/SessionControllerIndex.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.PlatformAbstractions;
+using TestingControllersSample;
+using Xunit;
+
+namespace TestingControllerSample.Tests.IntegrationTests
+{
+    public class SessionControllerIndex
```

Comment:
That style, while more consistent with the product team guidance, tends to result in more bloated and hard to follow test classes. Hence my preference for more focused classes that better follow SRP. Are you certain you need this change made?